source/Public/New-GuestConfigurationPolicy.ps1

```diff
@@ -221,6 +257,11 @@ function New-GuestConfigurationPolicy
         throw "The specified package URI does not follow the HTTP or HTTPS scheme. Please specify a valid HTTP or HTTPS URI with the ContentUri parameter."
     }
 
+    if ($PSCmdlet.ParameterSetName -eq 'ManagedIdentity' -and ([string]::IsNullOrWhiteSpace($ManagedIdentityResourceId) -or [string]::IsNullOrWhiteSpace($LocalContentPath)))
+    {
+        throw "Both ManagedIdentityResourceId and LocalContentPath must be provided together."
```

Comment:
nit: I think this error message could be a little more helpful or descriptive? So that the user doesn't have to refer to the help section to fill in these values, but maybe would know what to fill in/fix from the message.

Reply:
Included the following message - 
Both ManagedIdentityResourceId and LocalContentPath must be provided together. Please include ManagedIdentityResourceId, LocalContentPath, and ExcludeArcMachines parameters.